src/osversion/test/3_9_1/test_vt_GroundHeatExchangerVertical.rb

```diff
@@ -0,0 +1,9 @@
+#require '/usr/local/openstudio-3.9.0/Ruby/openstudio'
+
+include OpenStudio::Model
+
+m = Model.new
+
+ghe_vert = GroundHeatExchangerVertical.new(m)
+
+m.save('test_vt_GroundHeatExchangerVertical.osm', true)
```

Comment:
(pedantic, but I kinda like the idea of hardsetting the before/after fields and the last one. Don't bother, just saying. It would avoid hidding some problems when the field was somehow reset to empty and the getString grabs the default value from IDD)